### PR TITLE
Fix/add sitellinks reflect tags mistake

### PR DESCRIPTION
**Pull request by William-Vu-1999**
**Add sitemap to robots.txt**

**add h1 tag if page missing h1**

**Write unique meta description for each pages in website**
> Meta description for PortfolioSubPage is overview
> Example: https://goldenowl.asia/portfolio/referral-candy
>image

> Meta description for ServicesSubPage is first content
> Example: https://goldenowl.asia/services/web-development
>image

### CODE REVIEW

File: src/pages/BlogDetails/index.js [hunk 1]
```diff
@@ -52,16 +52,10 @@ const BlogDetails = ({match}) => {
               content="width=device-width, initial-scale=1"
               name="viewport"
             />
-            {
```
Comment (austin-pham-goldenowl) [inline src/pages/BlogDetails/index.js:55]: Why remove this? What if the blog data is null?

File: src/pages/BlogDetails/index.js [hunk 2]
```diff
@@ -52,16 +52,10 @@ const BlogDetails = ({match}) => {
...
-                  <meta content={blog.attributes.title} property="og:title" />
-                  <meta content={blog.attributes.meta_description} property="og:description" />
-                  <meta content={companyOgLogo} property="og:image" />
-                </>
-              )
-            }
+            <meta content={blog.attributes.meta_description} name="description" />
+            <meta content={blog.attributes.title} property="og:title" />
+            <meta content={blog.attributes.meta_description} property="og:description" />
+            <meta content={companyOgLogo} property="og:image" />
```
Comment (austin-pham-goldenowl) [inline src/pages/BlogDetails/index.js:58]: Should replace the logo with thumbnail picture.

Comment (austin-pham-goldenowl) [reply src/pages/BlogDetails/index.js:58]: And also add <meta name="keywords" content="${blog.attributes.meta_keyword}">.

File: src/pages/BlogDetails/index.js (hunk 1 above)
Comment (William-Vu-1999) [reply src/pages/BlogDetails/index.js:55]: Data is already loaded, so it is unnescessary.
image

Comment (William-Vu-1999) [reply src/pages/BlogDetails/index.js:55]: And also, if using if else case here, the meta tags wont display

File: src/pages/BlogDetails/index.js (hunk 2 above)
Comment (William-Vu-1999) [reply src/pages/BlogDetails/index.js:58]: I have add `<meta name="keywords" content="${blog.attributes.meta_keyword}">`

**Pull request merged**